projects/wpdx/src/app/rehab-prio/rehab-prio.component.ts

```diff
@@ -56,7 +57,7 @@ export class RehabPrioComponent implements OnInit {
       filter(b => !!b),
       switchMap((bounds) => {
         const rehabPrio = this.db.query(this.queryUI(bounds));
-        const newConstructions = this.db.query(this.queryUINC(bounds));
+        const newConstructions = this.db.query(this.queryUINC(bounds, 15));
```

Comment:
Instead of using hardcoded `15` here, you could use the nc limit.
Take the last row for the minPopNC and then take the first 15 rows for the top10 variable.
You only need to call `this.updateState` after you get all results to make sure the map is updated.